Alignment/TrackerAlignment/plugins/TkAlCaOverlapTagger.cc

```diff
@@ -126,7 +126,7 @@ void TkAlCaOverlapTagger::produce(edm::Event& iEvent, const edm::EventSetup& iSe
 
       if ((previousTM != nullptr) && (layer != -1)) {
         for (std::vector<TrajectoryMeasurement>::const_iterator itmCompare = itTrajMeas - 1;
-             itmCompare >= tmColl.begin() && itmCompare > itTrajMeas - 4;
+             (itmCompare >= tmColl.begin()) && ((itmCompare > (itTrajMeas - 4)) > 0);
```

Comment:
Sorry, I don't understand this change: 

> ((itmCompare > (itTrajMeas - 4)) > 0)

 why is the boolean compared with 0?

Reply:
@aandvalenzuela , did you try 
```
(itmCompare >= tmColl.begin()) && ((itmCompare + 4) > itTrajMeas);
```

Reply:
agree with @[ChrisMisan](https://github.com/ChrisMisan) . either it should be `((itmCompare - (itTrajMeas - 4)) > 0)` or I think `((itmCompare + 4) > itTrajMeas)` might work. 

@aandvalenzuela , can you please try this?